Utils/WindowsSpecifics.go

```diff
@@ -19,99 +21,107 @@
  * under the License.
  ******************************************************************************/
 
-//go:build windows
-
 package Utils
 
 import (
 	"github.com/lxn/win"
 	"golang.org/x/sys/windows"
-	"os/exec"
-	"syscall"
 )
 
-/*
-RunningAsAdminPROCESSES checks if the program is running as administrator/root.
-
------------------------------------------------------------
+// == FUNCTION 1: RunningAsAdminPROCESSES ==
 
-– Returns:
-  - true if the program is running as admin, false otherwise
-*/
+/**
+ * Checks if the program is running with Administrator (root) privileges.
+ *
+ * @return true if running as admin, false otherwise
+ */
```

Comment:
Just a thing. Could you leave it with the documentation "format" I had? And on line 84+ too? I have it like that on purpose. If you click Ctrl+Q on GoLand with the cursor on the documentation, the documentation appears properly formatted. First I had as you put, but I noticed GoLand doesn't properly format it, so I changed the format to the one I use.

Reply:
I understand, I will make changes needed.